src/middlewared/middlewared/plugins/cluster_linux/ctdb_private.py

```diff
@@ -0,0 +1,241 @@
+from middlewared.schema import Dict, IPAddr
+from middlewared.service import (accepts, private, job,
+                                 Service, ValidationErrors,
+                                 CallError)
+from middlewared.plugins.cluster_linux.utils import CTDBConfig
+import pathlib
+
+
+SHARED_VOL = CTDBConfig.CTDB_LOCAL_MOUNT.value
+PRI_IP_FILE = CTDBConfig.GM_PRI_IP_FILE.value
+ETC_IP_FILE = CTDBConfig.ETC_PRI_IP_FILE.value
+PRI_LOCK = CTDBConfig.PRI_LOCK.value
+
+
+class CtdbIpService(Service):
+
+    class Config:
+        namespace = 'ctdb.private.ips'
+
+    @private
+    def query(self):
```

Comment:
It should be nice, even if being private, that it could accept the same arguments as every .query method we have. Or rename this method so its not confusing